Controllers/ProductController.js

```diff
@@ -43,6 +43,21 @@ const getSingleProduct = async(req, res) => {
     res.json(apiResponse)
 }
 
+const deleteSingleProduct = async(req, res) => {
+    const connection = await DbService()
+    const sku = DataCheckers.sanitiseSku(req.params.SKU)
+    const singleProduct = await ProductService.getSingleProduct(connection, sku)
+    const deleteProduct = await ProductService.deleteSingleProduct(connection, sku)
+    let apiResponse
+    if (singleProduct.length !== 0 && sku) {
```

Comment:
I think we should keep with this format for clarity and change the ternary syntax to this

Reply:
Do you mean change the ternaries on lines 10 and line 40 to be if statements instead?

Reply:
ternaries are gone!